source/connection_h1.c

```diff
@@ -445,7 +445,7 @@ struct aws_http_stream *s_new_client_request_stream(const struct aws_http_reques
 
     wrote_all &= aws_byte_buf_write_u8(&stream->outgoing_head_buf, '\r');
     wrote_all &= aws_byte_buf_write_u8(&stream->outgoing_head_buf, '\n');
-
+    (void)wrote_all;
```

Comment:
What does this do?

Reply:
It prevents a compiler warning that the value is never read when asserts are compiled out.